### PR TITLE
Updated Prim's MST

### PR DESCRIPTION
**Pull request by rap2montemayor**
Basically doing everything that was asked for in https://github.com/admu-progvar/progvar-library/issues/6
- Wrote a "not abridged" version in `pvl/graphs/mst/prim.hpp`
- Placed new code in `pvl/abridged/graphs/mst/prim.cpp`
- Added tests to `tests/graphs/mst/prim.cc`

I was also able to find a minor bug in `pvl/graphs/weighted_graphs.hpp`. The fix to it was included in this commit.

### CODE REVIEW

File: tests/graphs/CMakeLists.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 include_directories(shortest_paths)
-include_directories(max_flow)
\ No newline at end of file
+include_directories(max_flow)
+include_directories(mst)
```
Comment (leloykun) [inline tests/graphs/CMakeLists.txt:3]: [nit] add a new line

File: pvl/abridged/graphs/mst/prim.cpp [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
-      if (v == u)   continue;
-      if (vis[v])   continue;
-      res.push_back({w, {u, v}});
-      pq.push({w, v}); } } }
+    res.push_back(edge);
+    for (auto &[v, w] : adj[u])
+      if (!vis[v])
+        pq.push({w, {u, v}});
+  }
+}
```
Comment (leloykun) [inline pvl/abridged/graphs/mst/prim.cpp:20]: [nit] compress into a single line

we use the stuff in `pvl/abridged/` to generate the team notebooks so they need to be as short as possible

**Review by leloykun — CHANGES_REQUESTED** [review]
update the contributor's section in the README

**Comment by rap2montemayor** [comment]
Added all the requested changes. Should be ready to merge pull request.

**Review by leloykun — APPROVED** [review]

**Pull request merged**